mlir/include/mlir/Dialect/GEN/IR/GENDialect.td

```diff
@@ -20,28 +20,26 @@ def GEN_Dialect : Dialect {
     GEN is a dialect for representing operations on Intel GPUs.
   }];
 
-  let dependentDialects = ["mlir::LLVM::LLVMDialect"];
-
   let extraClassDeclaration = [{
     /// Get the name of the attribute used to annotate kernels.
     static StringRef getKernelFuncAttrName() { return "gen.kernel"; }
```

Comment:
I am not using `getKernelFuncAttrName` this in TritonGEN any longer. The kernel should be identified by having the SPIR_KERNEL calling convention so I don't think the dialect need to have this special function.

Reply:
Removed in the latest update, thanks.